unit-tests/live/d400/test-auto-limits.cpp

```diff
@@ -6,8 +6,6 @@
 /////////////////////////////////////////////////////////////////////////////
 
 //#test:device D400* !D457
-// auto-limits option is deprecated currently [LRS-487]
-//#test:donotrun
 
 #include "../../catch.h"
 #include "../../unit-tests-common.h"
```

Comment:
Please change
`//#test:device D400* !D457`
to specific D435i &&  D455 and remove the `are_limit_options_supported` logic.

This test will pass even if you forgot to register this option
Let's make sure this test actually run